src/Java.Interop/Java.Interop.targets

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <Runtime Condition="'$(OS)' != 'Windows_NT'">mono</Runtime>
   </PropertyGroup>
+  <Import Project="..\..\bin\BuildDebug\JdkInfo.props" Condition="Exists('..\..\bin\BuildDebug\JdkInfo.props')" />
```

Comment:
In what scenario will `JdkInfo.props` *not* exist? I don't understand why this `Condition` is required.

Also, this should be importing `Build$(Configuration)`, not `BuildDebug`.

Reply:
Ditto for all the other `<Import/>`s.